test/cppsuite/test_harness/core/configuration.h

```diff
@@ -37,7 +37,26 @@ extern "C" {
 #include "test_util.h"
 }
 
-enum class types { BOOL, INT, STRING, STRUCT };
+inline std::vector<std::string>
```

Comment:
I made this a free function because I'm going to use this to split the actual statistics strings up as we discussed.